tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -2074,7 +2089,7 @@ REGISTER_KERNEL_BUILDER(
                               .TypeConstraint<int64>("Tpaddings")          \
                               .Label(mkl_op_registry::kMklOpLabel),        \
                           MklConvOp<CPUDevice, float, float, float, float, \
-                                    float, int64, false, true, false>);    \
+                                    float, int32, false, true, false>);    \
```

Comment:
Since `padding_list` is of type `vector<int32>` in the layout pass, `Tpaddings` needs to be `int32` as well - otherwise the compiler complains about not being able to convert `int*` to `long long int*`  in line 1167:
```
1166 if (quantized_pad_enabled) {
1167  paddings = padding_list_.data();
1168} else {
```

Reply:
But then why do you need the type constraint 3 lines above to be `int64`? 
https://github.com/tensorflow/tensorflow/blob/0a507dc55b51e4f5651b57473ff5e2b3ea50d95e/tensorflow/core/kernels/mkl_conv_ops.cc#L2090